plugins/inputs/supervisor/supervisor.go

```diff
@@ -0,0 +1,183 @@
+package supervisor
+
+import (
+	"net/url"
+
+	"fmt"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/filter"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"github.com/kolo/xmlrpc"
+)
+
+type Supervisor struct {
+	Log telegraf.Logger `toml:"-"`
+
+	Server      string   `toml:"url"`
+	UseIdentTag bool     `toml:"use_identification_tag"`
+	MetricsInc  []string `toml:"metrics_include"`
+	MetricsExc  []string `toml:"metrics_exclude"`
+
+	rpcClient   *xmlrpc.Client
+	fieldFilter filter.Filter
+}
+
+type processInfo struct {
+	Name          string `xmlrpc:"name"`
+	Group         string `xmlrpc:"group"`
+	Description   string `xmlrpc:"description"`
+	Start         int32  `xmlrpc:"start"`
+	Stop          int32  `xmlrpc:"stop"`
+	Now           int32  `xmlrpc:"now"`
+	State         int16  `xmlrpc:"state"`
+	Statename     string `xmlrpc:"statename"`
+	StdoutLogfile string `xmlrpc:"stdout_logfile"`
+	StderrLogfile string `xmlrpc:"stderr_logfile"`
+	SpawnErr      string `xmlrpc:"spawnerr"`
+	ExitStatus    int8   `xmlrpc:"exitstatus"`
+	Pid           int32  `xmlrpc:"pid"`
+}
+
+type supervisorInfo struct {
+	StateCode int8   `xmlrpc:"statecode"`
+	StateName string `xmlrpc:"statename"`
+	Ident     string
+}
+
+const sampleConfig = `
+  ## Url of supervisor's XML-RPC endpoint if basic auth enabled in supervisor http server,
+  ## than you have to add credentials to url (ex. http://login:pass@localhost:9001/RPC2)
+  # url="http://localhost:9001/RPC2"
+  ## Use supervisor identification string as server tag
+  use_identification_tag = false
```

Comment:
I think this is a sensible default, so please use
```suggestion
  # use_identification_tag = false
```

Please check [the corresponding style documentation](https://github.com/influxdata/telegraf/blob/master/docs/developers/SAMPLE_CONFIG.md#style) for details...

Reply:
Included in upcoming commit